ansible/DockerfileConfigs

```diff
@@ -0,0 +1,16 @@
+FROM scratch
+
+COPY t /t
```

Comment:
Mentioned this to you at one point and it definitely shouldn't block anything, but if we want to save another ~MB (i.e., most of the size of these images), we could just do `FROM tianon/true`, which is a 125 byte image which contains and runs only a highly optimized x86 assembly version of `true`: https://github.com/tianon/dockerfiles/blob/master/true/true.asm